### PR TITLE
Appd links to obsolete version of website

### PR DESCRIPTION
**Pull request by mindthegab**

### CODE REVIEW

**Comment by mindthegab** [comment]
Actually I now wonder if that links was supposed to be updated to https://fdc3.finos.org/docs/1.0/appd-spec ?

File: src/app-directory/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Application Directory API -- THIS IS NOT A RATIFIED FDC3 API or SPECIFICATION
 
 **This is NOT a ratified API or specification but rather a tool to evaluate both the use cases and technology implementations.**
-[View FULL SPECIFICATION HERE](https://fdc3-app-directory.finos.org/)
+[View FULL FDC3 SPECIFICATION HERE](https://fdc3.finos.org/)
```
Comment (rikoe) [inline src/app-directory/README.md]: Ooh good catch, I never really looked at this file.

Yes it should probably point to https://fdc3.finos.org/docs/1.0/appd-spec.

File: src/app-directory/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Application Directory API -- THIS IS NOT A RATIFIED FDC3 API or SPECIFICATION
```
Comment (rikoe) [inline src/app-directory/README.md]: Should this text change now that is saying it is not a ratified specification?

File: src/app-directory/README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
 
 **This is NOT a ratified API or specification but rather a tool to evaluate both the use cases and technology implementations.**
-[View FULL SPECIFICATION HERE](https://fdc3-app-directory.finos.org/)
+[View FULL FDC3 SPECIFICATION HERE](https://fdc3.finos.org/)
 
 ## Overview
```
Comment (rikoe) [inline src/app-directory/README.md]: This overview is also slightly out of context now, should this specific README file be more about the details of the app-directory src files and code generation now @ftbb?

**Pull request merged**